atoma-proxy/src/server/components/openapi.rs

```diff
@@ -39,6 +43,7 @@ pub fn openapi_routes() -> Router {
             (path = CONFIDENTIAL_EMBEDDINGS_PATH, api = ConfidentialEmbeddingsOpenApi, tags = ["Confidential Embeddings"]),
             (path = IMAGE_GENERATIONS_PATH, api = ImageGenerationsOpenApi, tags = ["Images"]),
             (path = CONFIDENTIAL_IMAGE_GENERATIONS_PATH, api = ConfidentialImageGenerationsOpenApi, tags = ["Confidential Images"]),
+            (path = ENCRYPTION_PUBLIC_KEY_ENDPOINT, api = SelectNodePublicKeyOpenApi, tags = ["Node Public Key Selection"])
```

Comment:
maybe we should group some of them, now every endpoint has its own category (tag)
e.g. confidential would have all confidential stuff, including this endpoint
since it tagging and not a category maybe we can have category with confidential, and category with e.g. image which will also have confidential image

Reply:
I added a `Confidential` word to the tag for the public key encryption endpoint